frontend/src/metabase-lib/parameters/utils/click-behavior.ts

```diff
@@ -41,7 +44,7 @@ interface Target {
 }
 
 interface SourceFilters {
-  column: (column: DatasetColumn) => boolean;
+  column: (column: DatasetColumn, question?: Question) => boolean;
```

Comment:
Can we make it required?

Reply:
https://github.com/metabase/metabase/pull/37438/commits/aff38ff66570bf46a3851a6e926083b792893c49